src/overlays/actors/ovl_Obj_Toge/z_obj_toge.c

```diff
@@ -183,7 +183,10 @@ void ObjToge_Init(Actor* thisx, GlobalContext* globalCtx) {
 }
 
 void ObjToge_Destroy(Actor* thisx, GlobalContext* globalCtx) {
-    Collider_DestroyCylinder(globalCtx, &THIS->collider);
+    GlobalContext* globalCtx = globalCtx2;
```

Comment:
Jenkins is mad because it's still `void ObjToge_Destroy(Actor* thisx, GlobalContext* globalCtx)`, Needs to be `globalCtx2` for this to work.